Descent3/gamesequence.cpp

```diff
@@ -1401,6 +1401,7 @@ bool DoLevelIntro() {
 #endif
   //	if this level has a movie:
   if (lvl->flags & LVLFLAG_STARTMOVIE) {
+    SetScreenMode(SM_CINEMATIC);
```

Comment:
Probably we need this for `LVLFLAG_ENDMOVIE` too?

Reply:
Indeed, good catch!
Glad we have <DEL+E> to end level instantly. Tested on Retribution Level 15 where this happened as well.
Moving the `SetScreenMode()` call into `DoMissionMovie()` to fix it.